### PR TITLE
[EXP-39-240] Add id 6 to curve Ids enum

### PR DESCRIPTION
**Pull request by BobTheBuidler**

### CODE REVIEW

File: yearn/prices/curve.py [hunk 1]
```diff
@@ -60,7 +60,9 @@
         'voting_escrow': '0x5f3b5DfEb7B28CDbD7FAba78963EE202a494e2A2',
         'gauge_controller': '0x2F50D538606Fa9EDD2B11E2446BEb18C9D5846bB',
         # additional factories not listed as metapool factory
-        'factories': ['0xF18056Bbd320E96A48e3Fbf8bC061322531aac99'],
```
Comment (banteg) [inline yearn/prices/curve.py:63]: maybe remove altogether, we can always add later and there is always a reference in history

File: yearn/prices/curve.py [hunk 2]
```diff
@@ -158,6 +161,18 @@ def load_factories(self):
...
+            for pool in pool_list:
+                if pool in self.factories[factory]:
+                    continue
+                # for curve v5 pools, pool and lp token are separate
+                lp_token = contract(factory).get_token(pool)
+                self.token_to_pool[lp_token] = pool
+                self.factories[factory].add(pool)
+
+
+        # we keep this so we can manually add factories that haven't yet been added to the on-chain address provider
```
Comment (banteg) [inline yearn/prices/curve.py:175]: this too

**Review by banteg — APPROVED** [review]

File: yearn/prices/curve.py (hunk 1 above)
Comment (BobTheBuidler) [reply yearn/prices/curve.py:63]: done

File: yearn/prices/curve.py (hunk 2 above)
Comment (BobTheBuidler) [reply yearn/prices/curve.py]: done

**Pull request merged**